### PR TITLE
Initial about.yml file creation

### PR DESCRIPTION
**Pull request by porta-antiporta**
Creating about.yml file for team api usage

### CODE REVIEW

File: .about.yml [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+licenses:
+  fec-eregs:
+    name: CC0
+    ural: https://github.com/18F/fec-eregs/blob/master/LICENSE.md
+
+# Who is the partner for your project? (Use the full name of the partner
+# documented here:
+# https://github.com/18F/dashboard/blob/staging/_data/partners.yml)
+partners: Federal Election Commission
+-
```
Comment (adborden) [inline .about.yml]: This looks like a stray `-`

File: .about.yml [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+- github: annalee
+  role: developer
+- github: anthonygarvan
+  role: developer
+- github: edmullen
+  role: UX and Design
+- github: leahbannon
+- github: porta-antiporta
+  role: product manager
+- github: tadhg-ohiggins
```
Comment (adborden) [inline .about.yml]: Tadgh should have role: developer?

File: .about.yml [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+
+# What kind of content is contained in the project repository?
+# values: app, docs, policy
+type: app
+
+# What are the key milestones you've achieved recently?
+milestones:
+-
```
Comment (adborden) [inline .about.yml]: I'm not sure how this would be rendered in yaml, I would rewrite this as:

```
milestones: []
```

File: .about.yml [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+# values: app, docs, policy
+type: app
+
+# What are the key milestones you've achieved recently?
+milestones:
+- 
+
+# Name of the main project repo if this is a sub-repo; name of the grouplet
+# repo if this is a working group/guild subproject
+parent:
```
Comment (adborden) [inline .about.yml]: I think you could pick https://github.com/18F/fec for the parent repo.

File: .about.yml (hunk 3 above)
Comment (adborden) [inline .about.yml]: Okay, they're doing this all over the place so I guess it's fine.

**Comment by adborden** [comment]
How do you feel about removing the template comments and replacing it with a link to the template, so folks coming across the yaml know where the fields are coming from. Also by linking, the comments are never out of date, we'd always be linking to the latest version of the template.

**Comment by adborden** [comment]
Also, unless you plan on filling in the blank fields in the near future, I would remove them. I think the empty fields are just clutter and they're not helping to convey any information.

File: .about.yml (hunk 2 above)
Comment (porta-antiporta) [inline .about.yml]: Done.

**Pull request closed without merging**